nautobot/project-static/js/homepage_layout.js

```diff
@@ -0,0 +1,130 @@
+// Enables panels on homepage to be collapsed and expanded
+document.addEventListener('DOMContentLoaded', function() {
+    // Function to toggle and save state for a specific collapsible element
+    function toggleAndSaveState(elementId) {
+        // Remove 'toggle-' in the ID to get the localStorage key the toggle btn references
+        elementId = elementId.replace("toggle-", "");
+        var collapsibleDiv = document.getElementById(elementId);
+
+        // Toggle the collapsed class
+        if (collapsibleDiv.classList.contains('collapsed')) {
+            collapsibleDiv.classList.remove('collapsed');
+        } else {
+            collapsibleDiv.classList.add('collapsed');
+        }
```

Comment:
I think you could use `isCollapsed = collapsibleDiv.classList.toggle('collapsed')` here as a shorter alternative?

Reply:
Yup!